packages/core/theme-module/index.js

```diff
@@ -19,10 +19,18 @@ module.exports = async function DefaultThemeModule(moduleOptions) {
 
   const baseThemeDir = path.join(__dirname, 'theme');
   const projectLocalThemeDir = this.options.buildDir.replace('.nuxt', '.theme');
-  const themeComponentsDir = path.join(this.options.rootDir, 'components');
-  const themePagesDir = path.join(this.options.rootDir, 'pages');
-  const themeHelpersDir = path.join(this.options.rootDir, 'helpers');
-  const languageDir = path.join(this.options.rootDir, 'lang');
+
+  const getDirectoriesList = directoryPath => fs.readdirSync(directoryPath).filter(
```

Comment:
**code organization** We could put it in a separate file like `getAllFilesFromDir` and rename it to something that matches this pattern (like `getAllSubDirs`)